database/mssql/scripts/dbscripts/deploy.sh

```diff
@@ -0,0 +1,25 @@
+eval $(grep -v '^#' ../../.env | xargs)
```

Comment:
requires sqlcmd to be installed.
provides an easier way to run all of the scripts in a build/alter folder against a target db.